interface/patient_file/encounter/delete_form.php

```diff
@@ -82,7 +84,7 @@
 ?>
 <input type="hidden" id="confirm" name="confirm" value="1"/>
 <p>
-<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($_GET['formname'])); ?>
+<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text(getFormNameByFormdir($_REQUEST["formname"])["form_name"]); ?>
```

Comment:
Don't you need to translate it through the xl_form_title() function?

Reply:
Also, don't you just need to use $_GET['formname'] for formdir rather than $_REQUEST["formname"])["form_name"]

Reply:
1) Thanks  missed the translation after editing.
2) maybe this is more readable : 

$formdir=$_REQUEST["formname"];
$formName=getFormNameByFormdir($formdir);
echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($formName["form_name"]));

Dror G.

Reply:
P.S
I Saw in the code that sometimes there is a use of POST this is why I used REQUEST